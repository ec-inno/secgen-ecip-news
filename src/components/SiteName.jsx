import React from 'react';

const SiteName = () => {
  return (
    <div className="eci-name ecl-container ecl-u-pb-m ecl-u-type-m ecl-u-type-bold">
      <div>European</div>
      <div>Citizens'</div>
      <div>Initiative</div>
    </div>
  );
};

export default SiteName;
