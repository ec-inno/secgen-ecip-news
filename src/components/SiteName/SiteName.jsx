import React from 'react';
import { useTranslation } from 'react-i18next';

const SiteName = () => {
  const { t } = useTranslation();

  return (
    <div className="ecl-container ecl-u-pb-m ecl-u-type-xl ecl-u-type-bold">
      <span>{t('European')}</span>
      <span className="ecl-u-ml-2xs">{t("Citizens'")}</span>
      <span className="ecl-u-ml-2xs">{t('Initiative')}</span>
    </div>
  );
};

export default SiteName;
