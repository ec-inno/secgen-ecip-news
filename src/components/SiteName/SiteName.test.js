import React from 'react';
import renderer from 'react-test-renderer';
import SiteName from './SiteName';

describe('SiteName', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SiteName />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
