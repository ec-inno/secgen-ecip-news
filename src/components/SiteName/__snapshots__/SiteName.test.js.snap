// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteName renders correctly 1`] = `
<div
  className="ecl-container ecl-u-pb-m ecl-u-type-xl ecl-u-type-bold"
>
  <span>
    European
  </span>
  <span
    className="ecl-u-ml-2xs"
  >
    Citizens'
  </span>
  <span
    className="ecl-u-ml-2xs"
  >
    Initiative
  </span>
</div>
`;
